### DISCUSSION TITLE
Firebase phone auth do not working

### DISCUSSION THREAD

**Discussion by Hopheylalal** [firebase/flutterfire#5403] [Q&A]
my sha-1 and aha-256 debug and release in the app the same as in Firebase console. But i still have error in release mode

[firebase_auth/app-not-authorized] This app is not authorized to use Firebase Authentication. Please verify that the correct package name and SHA-1 are configured in the Firebase Console. [ App validation failed ]

in debug mode all is working

**Comment by vijaypareek13** [comment]
I also facing same problem

**Reply by vijaypareek13** [reply]
It is working with test number but not working with other number.  I added debug and release Sha1 but all in vain

**Comment by Pr47h4m** [comment]
Please follow up discussion  [#5492](https://github.com/FirebaseExtended/flutterfire/discussions/5492)
